include/bcd.h

```diff
@@ -238,11 +238,17 @@ const char *bcd_error_message(const bcd_error_t *);
 int bcd_error_errno(const bcd_error_t *);
 
 /*
- * Make key-value association.
+ * Make key-value association. KV-pairs are LIFO-ordered.
  */
 int bcd_kv(bcd_t *, const char *, const char *, bcd_error_t *);
 
 /*
+ * Add an argument to the command line. Arguments are presented to the tracing
+ * process in order.
+ */
+int bcd_arg(bcd_t *, const char *, bcd_error_t *);
```

Comment:
Argument order matters when calling into e.g. `coroner backtrace` and it feels unnatural to push arguments onto a stack that is applied in reverse order -- especially when the arguments are part of some configuration tunable. But either way, I think POLA applies. I don't see any reason to make argument order non-deterministic.